jest-preset.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
   },
   transformIgnorePatterns: ['node_modules/(?!@ngrx)'],
   snapshotSerializers: [
+    'jest-preset-angular/AngularSpecificAttributesRemover.js',
```

Comment:
Do you mind adding the suffix `Serializer` here?

Maybe let's also think about the naming of the serializer. It serializes Angular components as HTML, skipping `ng-*`-attributes. My suggestion therefore would not be to focus on the removal (which is the process the serializer does), but on the result, e. g. something like `AngularNoNgAttributesSnapshotSerializer.js

Reply:
Done